src/axom/doxygen_mainpage.md

```diff
@@ -14,6 +14,7 @@ Axom provides libraries that address common computer science needs.  It grew fro
 * @subpage primaltop provides an API for geometric primitives and computational geometry tests.
 * @subpage questtop provides an API to query point distance and position relative to meshes.
 * @subpage sidretop provides a data store with hierarchical structure.
+* @subpage sinatop provides an API to convert data to a common file format.
```

Comment:
> * @subpage sinatop provides an API to convert data to a common file format.

This feels a bit generic. Is there something that is still very high-level, but more specific we can say about `sina`?

Reply:
How does this sound: "The Sina ([S]mulation [In]sight and [A]nalysis) unified output library collects data directly within codes, outputting them to a common file output format co-designed with application developers and users"?